pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -2363,7 +2363,9 @@ type NetworkSpec struct {
 	// +optional
 	Connections *ConnectionsSpec `json:"connections,omitempty"`
 
-	// HostNetwork to enable host network
+	// HostNetwork to enable host network.
+	// If host networking is enabled or disabled on a running cluster, then the operator will automatically fail over all the mons to
+	// apply the new network settings.
 	// +optional
 	HostNetwork bool `json:"hostNetwork,omitempty"`
```

Comment:
The `hostNetwork` setting is no longer recommended, with `provider` being the preferred setting. I think perhaps we should be more clear about that by adding a new note, and move this recommendation up to that config.

Reply:
Keep it for both `network.HostNetwork` and `network.Provider`